molo/core/models.py

```diff
@@ -1441,7 +1441,10 @@ class ArticlePage(ImportableMixin, CommentedPageMixin,
         null=True)
     commenting_open_time = models.DateTimeField(null=True, blank=True)
     commenting_close_time = models.DateTimeField(null=True, blank=True)
-
+    commenting_anonymous_alias = models.CharField(
+        max_length=100,
+        blank=True,
+        default='anonymous')
```

Comment:
what happens on other languages that are not english? What does anonymous evaluate to? Does this get translated int he templates with {% trans %} tag?

Reply:
in molo-gem this is translated. will the tag work here? I wasnt sure about that

Reply:
@sewagodimo unless the template is changed to use the commenting_anonymous_alias, the current functionality won't change. What you would need to do is add something like {% trans commenting_anonymous_alias %} if it is the default

Reply:
Happy to jump on a call sometime tomorrow and step through it